crates/uv/tests/tool_upgrade.rs

```diff
@@ -126,21 +123,15 @@ fn test_tool_upgrade_all() {
 
     ----- stderr -----
     warning: `uv tool upgrade` is experimental and may change without warning
-    Resolved [N] packages in [TIME]
-    Prepared [N] packages in [TIME]
-    Uninstalled [N] packages in [TIME]
-    Installed [N] packages in [TIME]
+    Updated babel v2.6.0 -> v2.14.0
      - babel==2.6.0
      + babel==2.14.0
      - pytz==2018.5
-    Updated 1 executable: pybabel
-    Resolved [N] packages in [TIME]
-    Prepared [N] packages in [TIME]
-    Uninstalled [N] packages in [TIME]
-    Installed [N] packages in [TIME]
+    Installed 1 executable: pybabel
+    Updated python-dotenv v0.10.2.post2 -> v1.0.1
      - python-dotenv==0.10.2.post2
      + python-dotenv==1.0.1
```

Comment:
I wonder if we should _not_ show this for the target package...? Since it's already summarized above?

Reply:
I was on the fence in my suggestion, I don't have a strong preference. It sort of grounds the changes we're showing?

Reply:
Yeah I don't feel strongly. Seems ok to leave it for now.